WordPress/Classes/ViewRelated/Blog/Blogging Reminders/BloggingRemindersPushPromptViewController.swift

```diff
@@ -184,9 +184,9 @@ class BloggingRemindersPushPromptViewController: UIViewController {
             stackView.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: Metrics.edgeMargins.left),
             stackView.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -Metrics.edgeMargins.right),
             stackView.topAnchor.constraint(equalTo: view.topAnchor, constant: Metrics.edgeMargins.top),
-            stackView.bottomAnchor.constraint(lessThanOrEqualTo: turnOnNotificationsButton.topAnchor, constant: Metrics.edgeMargins.bottom),
 
-            turnOnNotificationsButton.heightAnchor.constraint(greaterThanOrEqualToConstant: Metrics.turnOnButtonHeight),
+            turnOnNotificationsButton.heightAnchor.constraint(equalToConstant: Metrics.turnOnButtonHeight),
```

Comment:
👋 @ipalm0423 - I was curious about this change for the button's height anchor from `greaterThanOrEqualToConstant` to `equalToConstant`.

Will this prevent the button height from growing with larger Dynamic Type settings?

Reply:
Yes, it will keep the same height when scaling up. 
But the button has `height == 44` and this height could handle most sizes in Dynamic type. 

• | iPhone | iPad
-- | -- | --
Max scale | ![Simulator Screen Shot - iPhone 14 - 2022-09-22 at 01 15 54](https://user-images.githubusercontent.com/10288494/191574571-854c5e2a-cfff-45bc-be93-0a0539d6d241.png) | ![Simulator Screen Shot - iPad (9th generation) - 2022-09-22 at 01 45 44](https://user-images.githubusercontent.com/10288494/191574732-664e6f9b-095c-4663-9b78-5ff6059465dd.png) |

Reply:
Thanks for the info @ipalm0423! My follow-up question to that would be:

What would be the downside of letting the button grow so that the padding is retained? I didn't do extensive testing, but this is what it looked like for me with `greaterThanOrEqualToConstant` and the max scale on iPad. In my opinion this looks better, but perhaps I'm missing something that now breaks when this is enabled? Thanks!

<img src="https://user-images.githubusercontent.com/2092798/191579333-6c666d0e-929e-465f-8bbc-03afb9b7259e.png" width="350px" />

Reply:
It's a trade-off between label and button space. I set the button to constant to give more space for the 3 labels in Dynamic Type. 

After checking the code, I found out the `hintLabel` will be [hidden](https://github.com/wordpress-mobile/WordPress-iOS/blob/trunk/WordPress/Classes/ViewRelated/Blog/Blogging%20Reminders/BloggingRemindersPushPromptViewController.swift#L156) when using DynamicType. 
It means the labels will have enough space to scale up. The button doesn't have to give them more.
So I modify my change and re-push it.
Thanks for your kind feedback, help me reconsider the best suitable solution🙏.


Here are my test results:

-- | Default | DynamicType |
-- | -- | --
iPad | ![ipad](https://user-images.githubusercontent.com/10288494/191697397-fac90c51-b741-4b15-9fa7-e8f18d915ebc.png) | ![ipad-d](https://user-images.githubusercontent.com/10288494/191697440-c2c5bfea-0c5c-4cae-88cd-b41dc89b1e1c.png)
iPhone | ![iphone](https://user-images.githubusercontent.com/10288494/191697849-a8d1ad3e-029f-4de0-9fc5-9adc38927f0e.png) | ![iphone-d](https://user-images.githubusercontent.com/10288494/191697895-b90b9769-8442-4b96-8e67-521e65c5678b.png)

Reply:
Thanks for the info!